wallaby.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = () => {
```

Comment:
See https://github.com/shelfio/base-dotfiles/blob/master/npm-open-source/wallaby.config.js 

There more changes needed in this file when switching to TS

Reply:
Done